tests/e2e/defaultvmgroup_test.go

```diff
@@ -171,3 +170,156 @@ func (s *DefaultVMGroupTestSuite) TestDeleteDefaultVmgroup(c *C) {
 
 	misc.LogTestEnd(c.TestName())
 }
+
+// Test steps:
```

Comment:
Please add a brief comment to explain the intention of this test. Same for the test below. Otherwise it's difficult for the reader to figure out what's the difference between TestMoveVMBetweenVmgroup1 and TestMoveVMBetweenVmgroup2.